### ISSUE TITLE
rpc error: code = DeadlineExceeded desc = context deadline exceeded

### ISSUE THREAD

**Issue by tobg** [piraeusdatastore/piraeus-operator#101]
I just did a re-installation of linstor on my 5 node k8s version 1.18.9 drbd 9.0.25.
It is not possible to create any pvc. I always get the error DeadlineExceeded.
Any solution? What can I do to debug this? Thanks.

This is the error from csi-plugin:

time="2020-10-13T11:32:42Z" level=error msg="method failed" func="github.com/sirupsen/logrus.(*Entry).Error" file="/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:297" error="rpc error: code = Internal desc = CreateVolume failed for pvc-a36918fc-0bb3-40ce-aace-ce86288d93d6: unable to determine volume topology: unable to determine AccessibleTopologies: context canceled" linstorCSIComponent=driver method=/csi.v1.Controller/CreateVolume nodeID=k8w2 provisioner=linstor.csi.linbit.com req="name:\"pvc-a36918fc-0bb3-40ce-aace-ce86288d93d6\" capacity_range:<required_bytes:1000000000 > volume_capabilities:<mount:<fs_type:\"ext4\" > access_mode:<mode:SINGLE_NODE_WRITER > > parameters:<key:\"autoPlace\" value:\"1\" > parameters:<key:\"storagePool\" value:\"lvm-hdd\" > " resp= version=v0.9.0

**Comment by WanzenBug** [comment]
Looks like CSI is times out when waiting for the LINSTOR controller. Please also provide us the output of
* `kubectl exec -it deployment/<LINSTOR-Controller-deployment> -- linstor node list`
* `kubectl exec -it deployment/<LINSTOR-Controller-deployment> -- linstor error-reports list`
* `kubectl exec -it deployment/<LINSTOR-Controller-deployment> -- linstor resource list`
Note: you have to replace `<LINSTOR-Controller-deployment>` with your own value. In our example from the README it would be `piraeus-op-cs-controller`, but you might use a different name.

**Comment by tobg** [comment]
Linstor node list:

 Node                                      ┊ NodeType   ┊ Addresses                    ┊ State  ┊
╞════════════════════════════════════════════════════════════════════════════════════════════════╡
┊ k8smaster                                 ┊ SATELLITE  ┊ 165.22.30.49:3366 (PLAIN)    ┊ Online ┊
┊ k8w2                                      ┊ SATELLITE  ┊ 94.237.48.153:3366 (PLAIN)   ┊ Online ┊
┊ k8w3                                      ┊ SATELLITE  ┊ 94.237.118.185:3366 (PLAIN)  ┊ Online ┊
┊ k8w4                                      ┊ SATELLITE  ┊ 94.237.9.94:3366 (PLAIN)     ┊ Online ┊
┊ piraeus-op-cs-controller-747f84c986-sf56f ┊ CONTROLLER ┊ 192.168.246.152:3366 (PLAIN) ┊ Online ┊

error-report list:

No list

resource list:

╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
┊ ResourceName                             ┊ Node      ┊ Port ┊ Usage  ┊ Conns                                  ┊        State ┊ CreatedOn           ┊
╞════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╡
┊ pvc-0df443e8-076e-442c-ae37-3370a5b7b497 ┊ k8w2      ┊ 7013 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-0df443e8-076e-442c-ae37-3370a5b7b497 ┊ k8w4      ┊ 7013 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-0e8dcee6-bed0-4647-9351-977f6fd4a9e0 ┊ k8smaster ┊ 7085 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 19:29:58 ┊
┊ pvc-1d842595-1c30-4e7a-a22e-8948deafa242 ┊ k8smaster ┊ 7062 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-07 19:31:25 ┊
┊ pvc-2aa0c41b-8a17-4b02-b234-b532eba4be4d ┊ k8smaster ┊ 7009 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-3b177ed6-e7c1-477a-b92f-ffdc98c6e1b5 ┊ k8w2      ┊ 7011 ┊ Unused ┊ Connecting(k8w4)                       ┊     Outdated ┊                     ┊
┊ pvc-3b177ed6-e7c1-477a-b92f-ffdc98c6e1b5 ┊ k8w4      ┊ 7011 ┊ Unused ┊ StandAlone(k8w2)                       ┊     UpToDate ┊                     ┊
┊ pvc-3baf9efe-92a4-4ccd-a16f-8729b186e161 ┊ k8w2      ┊ 7000 ┊ Unused ┊ Connecting(k8w4)                       ┊     Outdated ┊                     ┊
┊ pvc-3baf9efe-92a4-4ccd-a16f-8729b186e161 ┊ k8w4      ┊ 7000 ┊ Unused ┊ StandAlone(k8w2)                       ┊ Inconsistent ┊                     ┊
┊ pvc-3f9a8432-3f9e-407e-98d3-bdd623a8ab16 ┊ k8smaster ┊ 7025 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-5af7cc30-72ba-425e-ad7b-8106bf608ed8 ┊ k8w2      ┊ 7088 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 21:19:55 ┊
┊ pvc-5af7cc30-72ba-425e-ad7b-8106bf608ed8 ┊ k8w4      ┊ 7088 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 21:18:53 ┊
┊ pvc-05b6b681-99b3-44d0-8f75-98bbe1767250 ┊ k8smaster ┊ 7053 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-02 16:04:19 ┊
┊ pvc-5b7ad005-08e4-4bc4-8fc9-aa4c040e20a3 ┊ k8smaster ┊ 7031 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-6b2b50df-a5b9-4d80-84fa-57e532266916 ┊ k8smaster ┊ 7080 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-11 18:35:07 ┊
┊ pvc-6e51bda3-eccc-44de-bd42-b7e0ed03beea ┊ k8smaster ┊ 7026 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-6fb45624-ff7b-4a63-b07b-dab10bd03c17 ┊ k8w2      ┊ 7002 ┊ Unused ┊ Ok                                     ┊     Diskless ┊                     ┊
┊ pvc-6fb45624-ff7b-4a63-b07b-dab10bd03c17 ┊ k8w4      ┊ 7002 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-7c71a0c8-cce9-4ff8-8f98-fee1205e8652 ┊ k8smaster ┊ 7070 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-09 14:52:25 ┊
┊ pvc-7e931da6-6a8d-431c-b850-d7cd8054ba1c ┊ k8w2      ┊ 7087 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 20:50:41 ┊
┊ pvc-7e931da6-6a8d-431c-b850-d7cd8054ba1c ┊ k8w4      ┊ 7087 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 20:49:23 ┊
┊ pvc-8dd226e2-bdb8-4638-9476-438989340127 ┊ k8smaster ┊ 7084 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 19:08:12 ┊
┊ pvc-8df5112a-2b89-44f9-9cf2-4149acef4159 ┊ k8smaster ┊ 7010 ┊ InUse  ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-8e2fc016-75f2-4a99-a20c-884d33e0fd22 ┊ k8smaster ┊ 7069 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-09 13:01:56 ┊
┊ pvc-8e8d0167-3e50-4d64-ae26-39e7489d73c8 ┊ k8smaster ┊ 7054 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-03 20:03:27 ┊
┊ pvc-9af4bd2b-360f-4ca3-adbf-f330795db072 ┊ k8w2      ┊ 7043 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-09-29 21:56:47 ┊
┊ pvc-9af4bd2b-360f-4ca3-adbf-f330795db072 ┊ k8w4      ┊ 7043 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-09-29 21:56:57 ┊
┊ pvc-9b8c2d76-11b1-4fcf-9d85-b4239c9d893f ┊ k8smaster ┊ 7006 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-09e96d41-ce90-4e7e-9292-b68e9e748e65 ┊ k8smaster ┊ 7065 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-07 20:30:21 ┊
┊ pvc-9efff340-b88e-4053-b2bc-1e7e8e6c4a35 ┊ k8w2      ┊ 7095 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 09:41:25 ┊
┊ pvc-11ca1a18-6913-4c4c-a9df-3663880d3c03 ┊ k8smaster ┊ 7075 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-10 13:09:37 ┊
┊ pvc-14e4b5e9-9994-43af-8bd7-05444a80053f ┊ k8smaster ┊ 7066 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-07 21:31:09 ┊
┊ pvc-17e32d55-d74e-41ef-9df6-e86d32719828 ┊ k8w2      ┊ 7017 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-17e32d55-d74e-41ef-9df6-e86d32719828 ┊ k8w4      ┊ 7017 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-34f6d697-e9df-4761-b7e9-1c311e7d68d7 ┊ k8w2      ┊ 7018 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-34f6d697-e9df-4761-b7e9-1c311e7d68d7 ┊ k8w4      ┊ 7018 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-43f52219-c693-490c-bc49-278ae5fc7df1 ┊ k8smaster ┊ 7028 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-45a2e4f7-fdbb-471a-9247-c33a6140491c ┊ k8w2      ┊ 7042 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-09-30 16:50:54 ┊
┊ pvc-45a2e4f7-fdbb-471a-9247-c33a6140491c ┊ k8w4      ┊ 7042 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-09-30 16:51:06 ┊
┊ pvc-46da487d-139d-4e45-a530-4f6c9df7e5e7 ┊ k8smaster ┊ 7098 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 10:45:05 ┊
┊ pvc-46da487d-139d-4e45-a530-4f6c9df7e5e7 ┊ k8w2      ┊ 7098 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 10:48:24 ┊
┊ pvc-46da487d-139d-4e45-a530-4f6c9df7e5e7 ┊ k8w3      ┊ 7098 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 10:45:05 ┊
┊ pvc-46da487d-139d-4e45-a530-4f6c9df7e5e7 ┊ k8w4      ┊ 7098 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 10:46:21 ┊
┊ pvc-54f6f0c6-f9c4-46f9-a3d9-2f7066f358a5 ┊ k8smaster ┊ 7046 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-01 12:08:38 ┊
┊ pvc-57c96687-da3e-42da-a4bc-ebf742abb8e9 ┊ k8smaster ┊ 7067 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-09 12:17:43 ┊
┊ pvc-60be9b00-f476-420b-bc07-a6e87f972bb1 ┊ k8w2      ┊ 7037 ┊ Unused ┊ Connecting(k8w4)                       ┊     Outdated ┊ 2020-10-02 17:41:16 ┊
┊ pvc-60be9b00-f476-420b-bc07-a6e87f972bb1 ┊ k8w3      ┊ 7037 ┊ Unused ┊ StandAlone(k8w4)                       ┊     Outdated ┊ 2020-10-02 17:41:16 ┊
┊ pvc-60be9b00-f476-420b-bc07-a6e87f972bb1 ┊ k8w4      ┊ 7037 ┊ InUse  ┊ StandAlone(k8w3,k8w2)                  ┊     UpToDate ┊ 2020-10-02 17:41:16 ┊
┊ pvc-68ee2aa2-5e80-416a-bce3-b5e6b8ed47ca ┊ k8smaster ┊ 7073 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-10 11:51:58 ┊
┊ pvc-74d1511c-0a8c-4bc2-a75d-35d73b1a589a ┊ k8w2      ┊ 7096 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 09:51:25 ┊
┊ pvc-83c4f172-2706-45bb-b7f6-4f7ebf4f2d83 ┊ k8smaster ┊ 7038 ┊ Unused ┊ StandAlone(k8w2)                       ┊     Outdated ┊                     ┊
┊ pvc-83c4f172-2706-45bb-b7f6-4f7ebf4f2d83 ┊ k8w2      ┊ 7038 ┊ Unused ┊ Connecting(k8w4),StandAlone(k8smaster) ┊     Outdated ┊                     ┊
┊ pvc-83c4f172-2706-45bb-b7f6-4f7ebf4f2d83 ┊ k8w4      ┊ 7038 ┊ InUse  ┊ StandAlone(k8w2),Connecting(k8smaster) ┊     UpToDate ┊                     ┊
┊ pvc-96f6e4e3-3f75-4fac-901f-070c9a9a1eb4 ┊ k8w2      ┊ 7050 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-02 00:16:35 ┊
┊ pvc-96f6e4e3-3f75-4fac-901f-070c9a9a1eb4 ┊ k8w4      ┊ 7050 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-02 00:16:30 ┊
┊ pvc-137feb15-a995-42f2-be65-469c861556d1 ┊ k8w2      ┊ 7097 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 10:25:50 ┊
┊ pvc-141dc842-13a7-4a3b-824d-2518a6cf6625 ┊ k8smaster ┊ 7016 ┊ Unused ┊ Ok                                     ┊     Diskless ┊                     ┊
┊ pvc-141dc842-13a7-4a3b-824d-2518a6cf6625 ┊ k8w2      ┊ 7016 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-141dc842-13a7-4a3b-824d-2518a6cf6625 ┊ k8w4      ┊ 7016 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-307c019f-2dfc-4f63-9baf-fd116c5f2bad ┊ k8smaster ┊ 7059 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-07 18:29:38 ┊
┊ pvc-424c049f-f639-4c07-9feb-a588acaf8127 ┊ k8smaster ┊ 7052 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-02 15:18:45 ┊
┊ pvc-499ab1e4-c7b1-4720-b54b-7cc590cd83a2 ┊ k8smaster ┊ 7051 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-02 00:23:47 ┊
┊ pvc-519b1fad-0445-4ec3-a3d9-a5276150f723 ┊ k8smaster ┊ 7071 ┊ InUse  ┊ Ok                                     ┊     UpToDate ┊ 2020-10-09 17:18:17 ┊
┊ pvc-523b9e1c-b5d2-4931-9834-1c72e572f074 ┊ k8smaster ┊ 7001 ┊ Unused ┊ Ok                                     ┊     Diskless ┊                     ┊
┊ pvc-523b9e1c-b5d2-4931-9834-1c72e572f074 ┊ k8w2      ┊ 7001 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-523b9e1c-b5d2-4931-9834-1c72e572f074 ┊ k8w4      ┊ 7001 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-553e52ac-2e5e-4e7e-9d5a-8d27dd1fa934 ┊ k8w2      ┊ 7092 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 08:08:53 ┊
┊ pvc-592ad756-2798-487e-b9a0-d5a2351628e9 ┊ k8smaster ┊ 7022 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-694d2078-61ce-4c3d-ac51-4e9a98be7871 ┊ k8smaster ┊ 7083 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 18:23:13 ┊
┊ pvc-962fe9d4-f5b7-4f58-80ed-a3123c761b20 ┊ k8smaster ┊ 7007 ┊ InUse  ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-4192eb5a-4753-452c-9e58-ecf1b2581769 ┊ k8smaster ┊ 7064 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-07 20:14:44 ┊
┊ pvc-6594b913-027f-45c3-9ce8-63896d81510b ┊ k8smaster ┊ 7063 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-07 20:14:38 ┊
┊ pvc-7009e966-593c-48aa-9de6-0a96a5fff218 ┊ k8smaster ┊ 7074 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-10 11:52:04 ┊
┊ pvc-7833d396-3025-46d0-b354-237fb9bb6e3d ┊ k8smaster ┊ 7045 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-01 12:04:07 ┊
┊ pvc-19503ae8-b383-473b-bb3e-4fcfc8324319 ┊ k8w2      ┊ 7019 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-19503ae8-b383-473b-bb3e-4fcfc8324319 ┊ k8w4      ┊ 7019 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-59291e8b-12c7-4930-9f43-79e62de97e53 ┊ k8smaster ┊ 7058 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-07 18:27:00 ┊
┊ pvc-073410a5-8b59-4c67-baea-9c95abe53c85 ┊ k8w2      ┊ 7015 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-073410a5-8b59-4c67-baea-9c95abe53c85 ┊ k8w4      ┊ 7015 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-4351402e-5c0d-4aab-88e7-e2a1ea72609e ┊ k8smaster ┊ 7023 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-8487939d-a2dc-4ffb-b18a-5260bc382f97 ┊ k8w2      ┊ 7090 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 21:42:47 ┊
┊ pvc-8487939d-a2dc-4ffb-b18a-5260bc382f97 ┊ k8w4      ┊ 7090 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 21:41:01 ┊
┊ pvc-35093222-aefd-4830-b970-2b87e3b24c0d ┊ k8smaster ┊ 7004 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-a85bbf80-b602-4e35-baea-361086b8ccb9 ┊ k8w3      ┊ 7086 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 20:40:27 ┊
┊ pvc-a366bfd1-60a5-4c05-909c-d5a332f8a548 ┊ k8smaster ┊ 7079 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-11 18:31:54 ┊
┊ pvc-a36918fc-0bb3-40ce-aace-ce86288d93d6 ┊ k8smaster ┊ 7099 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 11:23:57 ┊
┊ pvc-a4970419-4f2d-4da4-8f85-b76ba5782843 ┊ k8w2      ┊ 7094 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 09:21:04 ┊
┊ pvc-ac82a5b5-bd66-45c8-8f62-3116fdea3185 ┊ k8smaster ┊ 7005 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-afc8f321-368c-42af-bf02-b7dd2eb01ce9 ┊ k8smaster ┊ 7082 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 18:13:25 ┊
┊ pvc-b01c4bc5-02a9-45d2-981c-d78f67bf613d ┊ k8w4      ┊ 7072 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-10 09:31:05 ┊
┊ pvc-b1c33370-307b-4de6-a8e7-0ff7eda01f69 ┊ k8w2      ┊ 7014 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-b1c33370-307b-4de6-a8e7-0ff7eda01f69 ┊ k8w4      ┊ 7014 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-b38c1ae7-4548-4624-889c-61a8ca65c764 ┊ k8smaster ┊ 7100 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 12:10:39 ┊
┊ pvc-b499abb2-260e-4815-92f1-efb827428b6c ┊ k8smaster ┊ 7024 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-b749840a-19e3-4981-8355-7e8d0677cfdc ┊ k8smaster ┊ 7003 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-ba4d4e9a-fc0a-4413-850c-8e3e68c93799 ┊ k8smaster ┊ 7068 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-09 12:59:42 ┊
┊ pvc-c36f750d-357c-4347-a7ac-832335ae2758 ┊ k8smaster ┊ 7029 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-c6587652-2a4a-4fd5-853c-2db9be59fc17 ┊ k8w2      ┊ 7093 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 08:31:38 ┊
┊ pvc-cd94761d-bd18-4d50-821d-f9897f4072ba ┊ k8smaster ┊ 7056 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-03 21:57:10 ┊
┊ pvc-d91fb292-c5cb-4bfe-a7b0-464a4dddb200 ┊ k8smaster ┊ 7048 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-01 23:04:28 ┊
┊ pvc-d834d857-3efb-432b-8e6e-482b1b508266 ┊ k8w2      ┊ 7077 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-11 15:26:44 ┊
┊ pvc-d834d857-3efb-432b-8e6e-482b1b508266 ┊ k8w4      ┊ 7077 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-11 15:26:44 ┊
┊ pvc-d501232a-d2eb-4d09-af92-1b75f5446251 ┊ k8smaster ┊ 7060 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-07 18:37:32 ┊
┊ pvc-dc886c4a-5a4e-4b2b-b9d9-ea34f5452a76 ┊ k8smaster ┊ 7061 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-07 18:49:54 ┊
┊ pvc-dd3eb953-b9c9-48e8-85a2-5e0fb4f94aef ┊ k8smaster ┊ 7032 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-e3e363b0-e2ab-4438-9077-f11037b180b4 ┊ k8w2      ┊ 7081 ┊ Unused ┊ Ok                                     ┊     Diskless ┊ 2020-10-11 21:29:38 ┊
┊ pvc-e3e363b0-e2ab-4438-9077-f11037b180b4 ┊ k8w4      ┊ 7081 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-11 21:29:38 ┊
┊ pvc-e917a631-a710-4cd5-8662-8bf140005e6a ┊ k8smaster ┊ 7057 ┊ Unused ┊ StandAlone(k8w3,k8w2)                  ┊     Outdated ┊ 2020-10-06 16:23:07 ┊
┊ pvc-e917a631-a710-4cd5-8662-8bf140005e6a ┊ k8w2      ┊ 7057 ┊ Unused ┊ Connecting(k8w3),StandAlone(k8smaster) ┊     Outdated ┊ 2020-10-06 16:23:07 ┊
┊ pvc-e917a631-a710-4cd5-8662-8bf140005e6a ┊ k8w3      ┊ 7057 ┊ InUse  ┊ StandAlone(k8w2,k8w4,k8smaster)        ┊     UpToDate ┊ 2020-10-06 16:23:07 ┊
┊ pvc-e917a631-a710-4cd5-8662-8bf140005e6a ┊ k8w4      ┊ 7057 ┊ Unused ┊ StandAlone(k8w3)                       ┊ Inconsistent ┊ 2020-10-06 16:23:07 ┊
┊ pvc-e3292453-9182-4274-94b0-266bd3910fcf ┊ k8smaster ┊ 7008 ┊ InUse  ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-eb261871-9f1c-404c-be42-cdc27d2038dc ┊ k8w2      ┊ 7078 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-11 15:27:20 ┊
┊ pvc-eb261871-9f1c-404c-be42-cdc27d2038dc ┊ k8w4      ┊ 7078 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-11 15:27:21 ┊
┊ pvc-ebc3aa86-cd50-4ee1-8ea7-45a646d0d4f4 ┊ k8w2      ┊ 7047 ┊ Unused ┊ Connecting(k8w4)                       ┊     Outdated ┊ 2020-10-01 14:55:20 ┊
┊ pvc-ebc3aa86-cd50-4ee1-8ea7-45a646d0d4f4 ┊ k8w4      ┊ 7047 ┊ InUse  ┊ StandAlone(k8w2)                       ┊     UpToDate ┊ 2020-10-01 14:55:35 ┊
┊ pvc-ee523956-769b-4707-a680-5ca15bb3815a ┊ k8w2      ┊ 7091 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 06:12:46 ┊
┊ pvc-ee523956-769b-4707-a680-5ca15bb3815a ┊ k8w4      ┊ 7091 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-13 06:13:03 ┊
┊ pvc-f7bf9983-822d-4918-b862-2c1433cac8ee ┊ k8smaster ┊ 7036 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-f14cb884-ecf2-4ac5-b1ff-b0ed5be4cf37 ┊ k8w2      ┊ 7089 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 21:37:47 ┊
┊ pvc-f14cb884-ecf2-4ac5-b1ff-b0ed5be4cf37 ┊ k8w4      ┊ 7089 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-12 21:35:15 ┊
┊ pvc-f94b77f4-ba4f-4b1a-8ed4-52ab9015e1be ┊ k8smaster ┊ 7055 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-03 20:47:56 ┊
┊ pvc-f345a691-cd8c-45fa-aa09-0018a47cc334 ┊ k8smaster ┊ 7027 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊                     ┊
┊ pvc-fc7029ea-cb3c-44df-9ef7-6b9d474f5c3a ┊ k8w2      ┊ 7076 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-11 15:25:42 ┊
┊ pvc-fc7029ea-cb3c-44df-9ef7-6b9d474f5c3a ┊ k8w4      ┊ 7076 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-10-11 15:25:50 ┊
┊ pvc-fd922de7-d6df-4466-9f5c-28c0dd2a6f64 ┊ k8smaster ┊ 7040 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊ 2020-09-29 12:48:27 ┊
┊ pvc-fe1b8f19-d9ef-4301-b607-e9535499a5cd ┊ k8smaster ┊ 7030 ┊ Unused ┊ Ok                                     ┊     UpToDate ┊

**Comment by WanzenBug** [comment]
Interesting. It looks like CSI is hanging on the same thing that also slows the last list down. I will ask internally if someone has experienced this before.

**Comment by WanzenBug** [comment]
In the meantime could you please also check the logs of the satellite pods?

**Comment by tobg** [comment]
Checked all ns pods. only info messages. No errors

**Comment by tobg** [comment]
Just an update on this:

I got many errors like this in the kernel logs of the workers:

k8smaster kernel: [ 3000.907350] drbd pvc-e917a631-a710-4cd5-8662-8bf140005e6a/0 drbd1057 k8w3: ASSERTION __dec_unacked(peer_device) >= 0 FAILED in e_end_resync_block

**Comment by tobg** [comment]
Just another observation. In two nodes load average goes through the roof. Increasing since 3h. linstor v l command is now timed out every time. I have to hard reset my nodes to get it back to normal.

**Comment by WanzenBug** [comment]
Just to make sure DRBD is loaded correctly, please check the logs of the initContainer for all satellites. And you can also verify the loaded module version by checking `/sys/module/drbd/version`.

**Comment by tobg** [comment]
Thank you for all your support.

I was able to resolve the issue by let pods compile and inject drbd instead of have it directly installed on the worker nodes. I first uninstalled the drbd kernel module. After that, it started working again. But after view minutes two nodes get very high load as I described above. After I updated  all possible images (to a newer version) in values.yaml for helm, it now works as expected.

I now will compare my installation with the now available 1.1.0
